File: BasicsWithJava/ArtBookFragments/app/src/main/java/com/sumere/artbookfragments/roomdb/ArtDatabase.java
package com.sumere.artbookfragments.roomdb;

import androidx.room.Database;
import androidx.room.RoomDatabase;
import com.sumere.artbookfragments.model.Art;

@Database(entities = {Art.class},version = 1)
public abstract class ArtDatabase extends RoomDatabase{
    public abstract ArtDao artDao();
}

File: ArtBookFragments/app/src/main/java/com/sumere/artbookfragments/roomdb/ArtDatabase.kt
package com.sumere.artbookfragments.roomdb

import androidx.room.Database
import androidx.room.RoomDatabase
import com.sumere.artbookfragments.model.Art

@Database(entities = arrayOf(Art::class), version = 1)
abstract class ArtDatabase: RoomDatabase() {
    abstract fun artDao(): ArtDao
}